backend/services/team_service.py

```diff
@@ -561,6 +561,7 @@ def send_message_to_all_team_members(
                     message.message_type = MessageType.TEAM_BROADCAST.value
                     message.save()
                     user = UserService.get_user_by_id(team_member.user_id)
-                    messages.append(dict(message=message, user=user))
+                    if user.teams_notifications:
```

Comment:
@ramyaragupathy I saw your last commit. Should we remove that check?

Reply:
Test locally with a team and flags on and off for users. Similar to mentions, DB messages are always sent, only emails updates are customizable by users.